test/controllers/IneligibleControllerSpec.scala

```diff
@@ -381,6 +381,29 @@ class IneligibleControllerSpec extends ItSpec {
       )
       ContentAssertions.commonIneligibilityTextCheck(page, TaxRegime.Sa, Languages.English)
     }
+    "SA generic ineligible page correctly for DmSpecialOfficeProcessingRequiredCESA eligibility reason" in {
```

Comment:
it'd be nice to have a test for this new flag in `DetermineEligibilityControllerSpec`- there's a big list of scenarios there that you can add to around here 
https://github.com/hmrc/essttp-frontend/blob/main/test/controllers/DetermineEligibilityControllerSpec.scala#L40